changelogs/unreleased/1888-remove-depracated-OptionVariable.yml

```diff
@@ -0,0 +1,7 @@
+---
+description: Remove support for leaving nullable attribute unassigned, an exception will now be raised. You should make sure optional variables are always assigned a value.
```

Comment:
We should probably also include something about the change of behavior of `OptionalValueException`, but that will have to wait until the discussion on Slack has reached a conclusion.